block/block/nft/nember_extract.py

```diff
@@ -0,0 +1,90 @@
+import argparse
+import json
+import pickle
+from binascii import unhexlify
+
+import msgpack
+from tqdm import tqdm
+
+from block.extractor import public_key_to_address
+
+
+def extract_nft_transfer(txn, height, nft_tx_map):
+    for mosaic in txn[b'payload'][b'mosaics']:
+        if mosaic[b'mosaic_id'] in nft_tx_map and mosaic[b'amount'] == 1:
+            nft_tx_map[mosaic[b'mosaic_id']].append((height, txn[b'payload'][b'recipient_address']))
+
+
+def main(args):
+    # pylint: disable=too-many-nested-blocks, too-many-branches
+
+    with open(args.input, 'rb') as file:
+        blocks = msgpack.unpack(file, unicode_errors=None, raw=True)
```

Comment:
I've tried running extract, but it fails for me during unpack - not sure what I did wrong:
```
raceback (most recent call last):
  File "block/nft/nember_extract.py", line 90, in <module>
    main(parsed_args)
  File "block/nft/nember_extract.py", line 22, in main
    blocks = msgpack.unpack(file, unicode_errors=None, raw=True)
  File "/usr/local/lib/python3.8/dist-packages/msgpack/__init__.py", line 58, in unpack
    return unpackb(data, **kwargs)
  File "msgpack/_unpacker.pyx", line 208, in msgpack._unpacker.unpackb
msgpack.exceptions.ExtraData: unpack(b) received extra data.
```